src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -863,7 +865,10 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 						// Go over all types we've seen
 						foreach (var value in methodParams[0].UniqueValues ()) {
 							if (value is SystemTypeValue systemTypeValue) {
-								MarkConstructorsOnType (ref reflectionContext, systemTypeValue.TypeRepresented, null, bindingFlags);
+								if (BindingFlagsAreSupported (bindingFlags) || unknownBindingFlags)
```

Comment:
I somehow missed this in the original PR - the method should be called `BindingFlagsAreUnsupported`